modules/core/src/main/java/org/apache/ignite/internal/processors/rest/protocols/tcp/GridTcpRestNioListener.java

```diff
@@ -145,6 +149,9 @@
     /** Handler for all Redis requests. */
     private GridRedisNioListener redisLsnr;
 
+    /** Client SSL enabled. */
+    private final boolean sslEnabled;
```

Comment:
isSslClientAuth could be a better name.

Reply:
I renamed field to `sslClientAuth`, because `is...` is a getter name, not field.